addons/mail/models/discuss/discuss_template.py

```diff
@@ -0,0 +1,63 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
+
+class DiscussTemplate(models.Model):
+    """Templates for sending Discuss messages."""
+
+    _name = "discuss.template"
```

Comment:
This name feels a bit too generic, shouldn't it be `discuss.channel.message.template` or something like that?